### DISCUSSION TITLE
Build is Queued

### DISCUSSION THREAD

**Discussion by emappstore777** [codemagic-ci-cd/codemagic-docs#2441] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

-

### Steps to reproduce

Hello, when try to start a new build - it does not build and goes to the queue. Is it a local problem or there are some issues on Code Magic side?

### Expected results

Hello, when try to start a new build - it does not build and goes to the queue. Is it a local problem or there are some issues on Code Magic side?

### Actual results

Hello, when try to start a new build - it does not build and goes to the queue. Is it a local problem or there are some issues on Code Magic side?

**Comment by mulderpf** [comment]
My build sat on the queue for a little while and then when the build started, it timed out while downloading Flutter.  I am back in the queue again it seems.

**Comment by helinanever** [comment, marked as answer]
This is related to the ongoing incident, please see the status page: https://status.codemagic.io/. Closing, as there is existing discussion related to this incident in https://github.com/orgs/codemagic-ci-cd/discussions/2424.

**Comment by Y4d4n-44** [comment]
i've similar problem, when i start a build, it goes directly to the queue, it's a Codemagic Platform problem, please try to inform it, thanks.

**Comment by karim-cardeo** [comment]
Builds are still stuck in queue, retried three times in the past hour.
